dist/Makefile

```diff
@@ -22,11 +22,6 @@ RIOTBASE ?= $(CURDIR)/../../RIOT
 # with the target all-valgrind instead of all:
 # make -B clean all-valgrind
 
-# Uncomment this to enable code in RIOT that does safety checking
```

Comment:
IMO this was useful and should be adapted to use the new DEVELHELP variable.

Reply:
I agree, but with the _new_ switch, `DEVELHELP` can enabled via env variable from everywhere, so no need to have this `comment out` option here. But I could leave a comment here, how to enable DEVELHELP, instead?

Reply:
I'd keep it as before, but no using ```#DEVELHELP = 1```